tests/arch/arm/arm_irq_vector_table/src/arm_irq_vector_table.c

```diff
@@ -188,33 +179,35 @@ void nrfx_power_clock_irq_handler(void);
 #if defined(CONFIG_BOARD_QEMU_CORTEX_M0)
 void timer0_nrf_isr(void);
 vth __irq_vector_table _irq_vector_table[] = {
-	nrfx_power_clock_irq_handler, 0, 0, 0, 0, 0, 0, 0,
-	timer0_nrf_isr, isr0, isr1, isr2
+	[POWER_CLOCK_IRQn]nrfx_power_clock_irq_handler,
+	[TIMER0_IRQn]timer0_nrf_isr,
+	[_ISR_OFFSET]isr0, isr1, isr2
 };
 #else
 vth __irq_vector_table _irq_vector_table[] = {
-	nrfx_power_clock_irq_handler,
-	isr0, isr1, isr2,
-	0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
-	rtc_nrf_isr
+	[POWER_CLOCK_IRQn]nrfx_power_clock_irq_handler,
+	[_ISR_OFFSET]isr0, isr1, isr2,
+	[RTC1_IRQn]rtc_nrf_isr,
```

Comment:
Why not use this as a common `_irq_vector_table` for all nRF series (including nRF54L after defining symbols for the system timer ISR and its interrupt number)?